meilisearch/models/client.py

```diff
@@ -0,0 +1,64 @@
+from datetime import datetime
+from typing import List, Optional
+
+from camel_converter.pydantic_base import CamelBase
+
+
+class _KeyBase(CamelBase):
+    uid: str
+    name: Optional[str] = None
+    description: str
+    actions: List[str]
+    indexes: List[str]
+    expires_at: Optional[datetime] = None
+
+    class Config:
+        json_encoders = {
+            datetime: lambda v: None
+            if not v
+            else f"{str(v).split('.', maxsplit=1)[0].replace(' ', 'T')}Z"
+        }
+
+
+class Key(_KeyBase):
+    key: str
+    created_at: datetime
+    updated_at: Optional[datetime] = None
+
+
+class KeyCreate(CamelBase):
+    name: Optional[str] = None
+    description: str
+    actions: List[str]
+    indexes: List[str]
+    expires_at: Optional[datetime] = None
+
+    class Config:
+        json_encoders = {
+            datetime: lambda v: None
+            if not v
+            else f"{str(v).split('.', maxsplit=1)[0].replace(' ', 'T')}Z"
+        }
```

Comment:
Where is this class used? Can we just use `Key` instead?

Reply:
You are right, I am using it for the `options` instead of a `dict`, but that isn't happening in this package. I'll remove it.